addons/google_account/models/google_service.py

```diff
@@ -23,6 +23,13 @@ class GoogleService(models.AbstractModel):
     _name = 'google.service'
     _description = 'Google Service'
 
+    def get_client_credentials(self, service):
```

Comment:
I think it has to be a private function

Reply:
I see what you mean, I hesitated on that one.

However chances are that in another module and other objects you will call get_client_credentials, so will look strange to make it private in these cases.

Reply:
for example here https://github.com/odoo/odoo/pull/64871/files#diff-f25dd53e60ee79027eeae83bc97ff0b42d687888c3fe59238b1411dbe355bbbbR24

Reply:
I meant for RPC calls

Reply:
thanks @kos94ok-3D , it is changed now